roles/validate/files/interface_configuration.yaml

```diff
@@ -19,10 +19,14 @@ $defs:
           - down
       untagged_vlan:
         type: integer
+        minimum: 0
+        maximum: 4096
```

Comment:
is this inclusive? because afaik we can't have VLAN ID 0 or 4096.

Reply:
I think you're right - I'll update those values